docs/providers.rst

```diff
@@ -760,9 +760,20 @@ fetching the access token::
 App registration (get your key and secret here)
     https://www.linkedin.com/secure/developer?newapp=
 
-Development callback URL
-    Leave the OAuth redirect URL empty.
+Authorized Redirect URLs (Oauth2)
```

Comment:
Sorry for nitpicking, but can you rename Oauth2 to OAuth2 (below as well), thanks.